src/state-summary/state-card-media_player.ts

```diff
@@ -0,0 +1,93 @@
+import type { HassEntity } from "home-assistant-js-websocket";
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import memoizeOne from "memoize-one";
+import { computeStateDisplay } from "../common/entity/compute_state_display";
+import "../components/entity/state-info";
+import HassMediaPlayerEntity from "../util/hass-media-player-model";
+import { HomeAssistant } from "../types";
+import { haStyle } from "../resources/styles";
+
+@customElement("state-card-media_player")
+class StateCardMediaPlayer extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj!: HassEntity;
+
+  @property({ type: Boolean }) public inDialog = false;
+
+  private _computePlayerObj = memoizeOne(
+    (hass: HomeAssistant, stateObj: HassEntity) =>
```

Comment:
This memoize is not really needed, as these are the only to properties that will cause an update of the element anyway, it will be recalculated every render.